snap/snapcraft.yaml

```diff
@@ -169,6 +175,8 @@ parts:
         # source only as trigger build whenever something changes in openHAB repo
         source: https://github.com/openhab/openhab-distro.git
         plugin: nil
+        # this source repository is used only as trigger for automated build
+        source: https://github.com/openhab/openhab-addons.git
```

Comment:
Isn't this a duplicate yaml key for the openhab plug?

Reply:
ha, it indeed does
fixing it